src/command_modules/azure-cli-acr/setup.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+
+from codecs import open
+from setuptools import setup
+
+VERSION = '0.0.5'
+
+CLASSIFIERS = [
+    'Development Status :: 3 - Alpha',
+    'Intended Audience :: Developers',
+    'Intended Audience :: System Administrators',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 2',
+    'Programming Language :: Python :: 2.7',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.4',
+    'Programming Language :: Python :: 3.5',
+    'License :: OSI Approved :: MIT License',
+]
+
+DEPENDENCIES = [
+    'azure==2.0.0rc6',
```

Comment:
Do you depend on the SDK, if so, you should use the specific components you need e.g. azure-mgmt-* instead.
Looks like that's why the build is failing with Keyvault errors as you're including an incompatible version of keyvault.